sdk/src/types/block/protocol.rs

```diff
@@ -20,27 +33,72 @@ use crate::types::block::{helper::network_name_to_id, output::RentStructure, Con
 )]
 pub struct ProtocolParameters {
     // The version of the protocol running.
-    #[serde(rename = "version")]
-    pub(crate) protocol_version: u8,
+    #[getset(get_copy = "pub")]
```

Comment:
` #[getset(get_copy = "pub")]` seems to be the most prominently used, just in case you are not aware you can set it on the struct itself and override, may be less verbose. Just ignore if it's been considered already.

Reply:
Yeah, it doesn't work because you can't do `skip_copy` for the fields that want the non-copy impl

Reply:
sad verbose noises